browser/resources/settings/brave_overrides/page_visibility.ts

```diff
@@ -53,6 +54,7 @@ function getPageVisibility () {
     // custom properties
     braveSync: !loadTimeData.getBoolean('isSyncDisabled'),
     braveWallet: loadTimeData.getBoolean('isBraveWalletAllowed'),
+    leoAssistant: !loadTimeData.getBoolean('isLeoAssistantDisabled'),
```

Comment:
I think you're only setting this if the `ENABLE_AI_CHAT` build flag is enabled. This will throw an error if that key doesn't exist in load time data (i.e if `ENABLE_AI_CHAT` is disabled)

Reply:
Changed `html_source->AddBoolean("isLeoAssistantDisabled",!ai_chat::features::IsAIChatEnabled());`  to be called always, not just when `ENABLE_AI_CHAT` is defined

https://github.com/brave/brave-core/pull/19212/commits/414fb6cd24fb8e0b6dfbf0eefabbcb9392065272#diff-38b6c12d746e933cd876120ea21df0a3ddc37a4879dbed20f364640ad3c60bdeR181